core/openapi/schemas/language.yaml

```diff
@@ -9,24 +9,24 @@ properties:
     description:
       The language tag as per RFC-5646.
     example: el
-    name:
-      type: string
-      minLength: 1
-      description:
-        The untranslated name of of the language.
-      example: Ελληνικά
-    alternate:
-      type: string
-      description:
-        The name of the language in another well-understood language,
-        usually English.
-      example: Greek
-    dir:
-      type: string
-      description:
-        The direction for text in this language.  Either left-to-right
-        or right-to-left.
-      enum:
-        - rtl
-        - ltr
-      default: ltr
+  name:
+    type: string
+    minLength: 1
+    description:
+      The untranslated name of of the language.
+    example: Ελληνικά
+  alternate:
+    type: string
+    description:
+      The name of the language in another well-understood language,
+      usually English.
+    example: Greek
+  dir:
+    type: string
+    description:
+      The direction for text in this language.  Either left-to-right
+      or right-to-left.  If this member is not specified then the assumed
```

Comment:
But why list it anyway then? You'll likely never encounter English in rtl. This was for informational purposes when you don't have a mapping in your client, so having it explicit helps to use the right direction without looking it up. I think as it is right now it could also be removed completely.

Reply:
@m-mohr I think the problem is that if someone specifies "Arabic" as the language but does not indicate a direction a default value of `ltr` would be assumes and that is not correct.  I have updated to the enum to add directions `ttb` ( to resolve [@cnreediii ](https://github.com/opengeospatial/ogcapi-records/pull/211#issuecomment-1495012337) comment), `btt` and `auto`.  The default is `auto` indicating that the client determines (somehow) the direction.  The value `btt` is added for completeness.  I don't know if there is an actual language that prints/displays `btt`; we can remove it later if it is useless.

Reply:
@m-mohr I'm going to merge this PR just to get it into the mainline.  If additional changes need to be made we can do that in another issue.